packages/cli-kit/src/public/node/error-handler.test.ts

```diff
@@ -22,13 +23,22 @@ vi.mock('@bugsnag/js', () => {
 vi.mock('./cli.js')
 vi.mock('./context/local.js')
 vi.mock('../../public/node/crypto.js')
+vi.mock('@oclif/core', () => ({
+  settings: {
+    debug: false,
+  },
+  Interfaces: {},
+}))
 
 beforeEach(() => {
   vi.mocked(ciPlatform).mockReturnValue({isCI: true, name: 'vitest', metadata: {}})
   vi.mocked(macAddress).mockResolvedValue('macAddress')
   vi.mocked(cloudEnvironment).mockReturnValue({platform: 'localhost', editor: false})
   vi.mocked(hashString).mockReturnValue('hashed-macaddress')
   vi.mocked(isUnitTest).mockReturnValue(true)
+  onNotify.mockClear()
+  delete process.env.SHOPIFY_SERVICE_ENV
```

Comment:
you are cleaning the state before, but not after, which means that this value will still be set after these tests finish, potentially affecting other tests.